dashboard/app/models/regional_partner.rb

```diff
@@ -195,7 +195,7 @@ def self.find_by_zip(zip_code)
             # Geocoder can raise a number of errors including SocketError, with a common base of StandardError
             # See https://github.com/alexreisner/geocoder#error-handling
             Retryable.retryable(on: StandardError) do
-              state = Geocoder.search({zip: zip_code})&.first&.state_code
+              state = Geocoder.search(zip_code)&.first&.state_code
```

Comment:
How was the `search(zip: zip_code)` syntax working?

We're on Geocoder 1.3.7.  Actually, we're on [Will's fork of Geocoder from August 2016](https://github.com/wjordan/geocoder/blob/rack-request-fix/lib/geocoder/version.rb).

https://github.com/code-dot-org/code-dot-org/blob/cb2de177ba1124871bd70a25c307e7d6c1c42bbb/Gemfile#L198-L199

(His fix has since been merged and we should probably switch back.  Geocoder's up to 1.5.1 now.)

But I don't see a mention of the old call pattern in either [the current docs](https://rdoc.info/github/alexreisner/geocoder/master/Geocoder#search-class_method) or [the docs for 1.3.7](https://github.com/alexreisner/geocoder/tree/v1.3.7). More reason to switch back, I guess. :+1:

Reply:
Same question about the 330 unmatched!